app/helpers/data_getter.py

```diff
@@ -216,16 +216,15 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
```

Comment:
@niranjan94 
For other uses where we need both `event_id` as well as `state` shouldn't we use `get_sessions_by_state_and_event_id` ( [https://github.com/fossasia/open-event-orga-server/blob/development/app/helpers/data_getter.py#L191](https://github.com/fossasia/open-event-orga-server/blob/development/app/helpers/data_getter.py#L191) ), I mean this function is being exclusively used for this particular function, isn't it ?